### PR TITLE
add curie mappings, prefix adjustments

### PR DESCRIPTION
**Pull request by kshefchek**
See https://github.com/biolink/biolink-model/issues/347

Adds curie mappings for unknowns, although I could not find mappings for the following:
 - VMC
 - SEMMEDDB
 - MYVARIANT_HG19
 - MYVARIANT_HG19
 - HGVS
 - CIViC (see https://github.com/griffithlab/civic-client/issues/1353)

I've made some casing changes as well (see WD -> wikidata) although I'm not sure if this will break a bunch of tools that rely on this.

### CODE REVIEW

**Comment by kshefchek** [comment]
fixes https://github.com/biolink/biolink-model/issues/365
fixes https://github.com/biolink/biolink-model/issues/358

**Comment by TomConlin** [comment]
+1

note: 
dots 
(or any punctuation except maybe hyphen if necessary) 
within curie-prefixes do not give me warm fuzzies.

**Comment by kshefchek** [comment]
yes the dots are not great, but this was to match the usage in identifiers.org, for example https://registry.identifiers.org/registry/iuphar.family, biolink could include some docs on how to handle these cases or best practices for prefixes.

**Comment by TomConlin** [comment]
It is very good to match existing usage where possible, 
However I will note identifiers.org is not really authoritative for anything 
and does not necessairly use in daily practice the items it traffics in. 
So it can more easily afford to be flexible.

File: biolink-model.yaml [hunk 1]
```diff
@@ -4,23 +4,37 @@ description: Entity and association taxonomy and datamodel for life-sciences dat
...
-  wgs: http://www.w3.org/2003/01/geo/wgs84_pos
-  qud: http://qudt.org/1.1/schema/qudt#
-  UMLSSG: https://uts-ws.nlm.nih.gov/rest/semantic-network/semantic-network/current/GROUP/
-  UMLSST: https://uts-ws.nlm.nih.gov/rest/semantic-network/semantic-network/current/STY/
-  UMLSSC: https://uts-ws.nlm.nih.gov/rest/semantic-network/semantic-network/current/TUI/
+  biolink: 'https://w3id.org/biolink/vocab/'
+  biolinkml: 'https://w3id.org/biolink/biolinkml/'
+  biosample: 'https://identifiers.org/biosample:'
+  CAID: 'http://reg.clinicalgenome.org/redmine/projects/registry/genboree_registry/by_caid?caid='
+  ECTO: 'http://purl.obolibrary.org/obo/ECTO_'
```
Comment (cmungall) [inline biolink-model.yaml:11]: we should get this from OBO, no major harm in duplicating, though DRY is good

File: biolink-model.yaml [hunk 2]
```diff
@@ -4,23 +4,37 @@ description: Entity and association taxonomy and datamodel for life-sciences dat
...
+  ExO: 'http://purl.obolibrary.org/obo/ExO_'
```
Comment (cmungall) [inline biolink-model.yaml:12]: we should get this from OBO, no major harm in duplicating, though DRY is good

File: biolink-model.yaml [hunk 3]
```diff
@@ -4,23 +4,37 @@ description: Entity and association taxonomy and datamodel for life-sciences dat
...
+  gtpo: 'https://rdf.guidetopharmacology.org/ns/gtpo#'
+  HANCESTRO: 'http://purl.obolibrary.org/obo/HANCESTRO_'
```
Comment (cmungall) [inline biolink-model.yaml:14]: we should get this from OBO, no major harm in duplicating, though DRY is good

File: biolink-model.yaml [hunk 4]
```diff
@@ -4,23 +4,37 @@ description: Entity and association taxonomy and datamodel for life-sciences dat
...
+  iuphar.family: 'https://identifiers.org/iuphar.family:'
+  intact: 'https://identifiers.org/intact:'
+  OBAN: 'http://purl.org/oban/'
+  pubchem.compound: 'https://identifiers.org/pubchem.compound:'
```
Comment (cmungall) [inline biolink-model.yaml:18]: doesn't this get mixed in from idot.context? no major harm in duplicating, though DRY is good

File: biolink-model.yaml (hunk 1 above)
Comment (kshefchek) [reply biolink-model.yaml:11]: this was on the list on unknowns but maybe the obo_prefix just needed a bump, I can remove

File: biolink-model.yaml (hunk 2 above)
Comment (kshefchek) [reply biolink-model.yaml:12]: same as above

File: biolink-model.yaml (hunk 3 above)
Comment (kshefchek) [reply biolink-model.yaml:14]: same as above

File: biolink-model.yaml (hunk 4 above)
Comment (kshefchek) [reply biolink-model.yaml:18]: i may have missed this when splitting out pubchem, I can remove

**Comment by kshefchek** [comment]
I just realized that idot == identifiers.org, so seems many of these we just need to align to that file (prefixes are all caps) and make sure it's updated

**Comment by kshefchek** [comment]
I've removed the entries for the obo and id.org prefixes, and converted all id.org to all uppercase to match the idot_context convention

**Comment by deepakunni3** [comment]
@kshefchek Should I go ahead and merge this?

**Pull request merged**